backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -103,15 +122,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // Match not found, insert participant entity into datastore
       Entity participantEntity = new Entity(KEY_PARTICIPANT);
       participantEntity.setProperty(PROPERTY_USERNAME, username);
-      participantEntity.setProperty(PROPERTY_STARTTIMEAVAILABLE, startTimeAvailable);
-      participantEntity.setProperty(PROPERTY_ENDTIMEAVAILABLE, endTimeAvailable);
+      participantEntity.setProperty(PROPERTY_START_TIME_AVAILABLE, startTimeAvailable.toString());
+      participantEntity.setProperty(PROPERTY_END_TIME_AVAILABLE, endTimeAvailable.toString());
       participantEntity.setProperty(PROPERTY_DURATION, duration);
       participantEntity.setProperty(PROPERTY_TIMESTAMP, timestamp);
       datastore.put(participantEntity);
```

Comment:
Optional (and maybe beyond the scope of this PR): We can match the if part by creating a function `addParticipantToDatastore`, where we create an entity and add it to the datastore.

Reply:
And we can pass the newParticipant to the function, to use the fields.

Reply:
Refactored this code to be part of its own method `addParticipanyEntityToDatastore`